src/app/compressed-air-assessment/explore-opportunities/explore-opportunities.component.html

```diff
@@ -26,6 +26,11 @@ <h3>Select Potential Adjustment Projects</h3>
                         <input name="modificatonName" type="text" id="modificatonName" class="form-control"
                             [(ngModel)]="modification.name" (input)="save()" />
                     </div>
+                    <div class="form-group group-name">
+                        <label for="flowReallocationImplementationCost">Flow Reallocation Implementation Cost</label>
+                        <input name="flowReallocationImplementationCost" type="text" id="flowReallocationImplementationCost" class="form-control"
```

Comment:
Needs to be of type number. Input is assigned as string, which is error prone. See JS doc string and number addition for why.